ZupIT/ritchie-cli
# Upgrade metric api

**- What I did**
I upgraded metrics API to improve security

**- How to verify it**
you can send a POST to `https://ritchie-metrics.devdennis.zup.io/v2/metrics` to see the authentication

**- Description for the changelog**
Upgrade metric API
